@uportal/content-carousel/src/components/ContentCarousel.html

```diff
@@ -8,7 +8,7 @@
   </template>
   <template v-else>
     <div class="slick-inner">
-      <slick ref="slick" :options="computedSlickOptions">
+      <slick ref="slick" :options="computedSlickOptions" :style="{ height }">
```

Comment:
Where does `height` come from?  Is it  property?  (Is it defined?)

If it's a property, I suggest it should work like...

`<content-carousel height="40rem">` and _not_ `<content-carousel height="height: 40rem">`

Reply:
To explain further, we already had an attribute called carousel-height that was not actually ever being used by the vue template. This fixes that by setting the height based on that attribute.